### PR TITLE
Homework2

### PR DESCRIPTION
**Pull request by smotan**

### CODE REVIEW

File: module2/homework/generateSequence/generateSequence.hpp [hunk 1]
```diff
@@ -2,6 +2,9 @@
 #include <vector>
 
 std::vector<int> generateSequence(int count, int step) {
-    // TODO: Implement me :)
-    return {};
+    std::vector<int> sequence;
```
Comment (piezol) [inline module2/homework/generateSequence/generateSequence.hpp:5]: You can preallocate memory here - size of the vector is known upfront.

File: module2/homework/generateSequence/generateSequence.hpp [hunk 2]
```diff
@@ -2,6 +2,9 @@
...
+    for(int i = 0; i < count; i++) {
+        sequence.push_back((i+1)*step);
```
Comment (Kasperice) [inline module2/homework/generateSequence/generateSequence.hpp:7]: Please use whitespaces around operators also here
```suggestion
        sequence.push_back((i + 1) * step);
```

File: module2/homework/generateSequence/generateSequence.hpp (hunk 1 above)
Comment (nauka-programowania-MA) [inline module2/homework/generateSequence/generateSequence.hpp:5]: use `resize` when you get knowledge of vector size.

**Comment by ziobron** [comment]
+4 XP addEven
+6 XP NwwNwd
+5 XP maxOfVector
+5 XP generateSequence
+6 XP for delivery before 31.05 for 3 tasks

{ "requirement_ids": [1119, 1631, 1119, 1633, 1119, 1634, 1635], "points": [2, 4, 2, 6, 2, 5, 5] }

**Pull request closed without merging**